webapp/src/app/shared/phaser/entities/entity-manager.ts

```diff
@@ -140,6 +145,12 @@ export class EntityManager extends BaseObject {
 					this.selectEntity();
 					this.showAddEntityMenu();
 				} else if (pointer.leftButtonReleased()) {
+
+					// if panning do not deselect entities
```

Comment:
@Vegita2 I think this would be an easy fix,  it would just mean putting this logic below the `isDragged = false`,  so that entities stop following the mouse

Reply:
@Vegita2 Thanks for providing the context.  Definitely helps understand whats going on and how to modify the behavior.  The place where this should go is right after `const entities = this.selectionBox.onInputUp();`, such that:


-  Selected entities will not be deselected when ctrl is released, and then ctrl is used for more panning
-  Pressing ctrl while dragging entities then releasing mouse will not cause entities to follow mouse when additional panning is performed.
entities won't follow the mouse when dragged, and the selection box will be removed.
-  Pressing ctrl in the middle of a selection box will remove the selection box once the mouse button is released.

Even though this has been approved, would like to make this modification, since its simple.

Reply:
Updated

Reply:
The problem with this is that the selection box now disappears, but the entities it selected are ignored. That means they are still in the hovered over state, but not actually selected. The hover state disappears when hovering over them again.

Reply:
Good catch.  Let me take another look

Reply:
@Vegita2 Tried a few different things, but can't seem to get it to work.  Want to avoid modifying the `selectEntity` method until I have a better understanding of what its doing, especially since its used in multiple places.  For now,  moved the return until after `isDragged = false`.  This brings back the issue where the selection box persists if ctrl is is pressed during the selection box.  How do you want to capture this defect,  should I post an issue on it?

Reply:
As i said i don't think this issue matters to much and we should just leave it as is. I don't think anyone ever tried to pan the map while simultaneously selecting entities.